app/views/cities/forbes_show.html.erb

```diff
@@ -30,11 +30,9 @@
   	<% end %>
     <% if @city.unapproved? %>
     <% else %>
-      <div class='sharing-info'>
-        <%= render partial: 'shared/sharing_widget', locals: {city: @city} %>
-      </div>
     <% end %>
     -->
+    <%= render partial: 'shared/sharing_widget', locals: {city: @city} %>
```

Comment:
let's get rid of this right now. we want your beautiful code but don't want to use it yet